plugins.json

```diff
@@ -4,28 +4,28 @@
         "package_name": "aiida",
         "entry_point": "",
         "state": "stable",
-        "pip_url": "git+https://github.com/aiidateam/aiida-core",
+        "pip_url": "aiida-core==1.0.0b6",
```

Comment:
Why are we freezing a given version of this package here? I would say we shouldn't, otherwise they'll be always outdated, am I right?

Reply:
as mentioned previously, this is a temporary fix for aiida-core.
the beta versions aren't released to `master` yet, so the `setup.json` is taken from the `1.0.0b6` tag.
this means, the `pip_url` should also work accordingly.

happy for better suggestions, in particular since we may face the same issue with other plugins in the next weeks.
of course, we could say we simply point to the `develop` branch and make the pip install instruction `--pre aiida-core`.

Reply:
Sorry, you are right, I had forgotten about those discussions. Indeed maybe develop would be nicer for other plugins. For aiida-core indeed ok as it is (maybe put a note in the aiida-core wiki on how to make releases to change this and then to remove the note from the wiki itself, so we don't forget).

Reply:
> maybe put a note in the aiida-core wiki on how to make releases to change this and then to remove the note from the wiki itself, so we don't forget

done!